service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocmosisTemplateDataGenerationTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.service.DocmosisTemplateDataGeneration.generateDraftWatermarkEncodedString;
+
+class DocmosisTemplateDataGenerationTest {
+
+    private static final DocmosisTemplateDataGeneration templateDataGeneration = new DocmosisTemplateDataGeneration() {
+        @Override
+        public Map<String, Object> getTemplateData(CaseData caseData) {
+            throw new NotImplementedException("Not testing this method");
+        }
+    };
+
+    @Test
+    void shouldGenerateEncodedStringWhenGenerateDraftWatermarkCalled() throws IOException {
+        String encodedImgString = generateDraftWatermarkEncodedString();
+        assertThat(encodedImgString).isNotEqualTo("");
```

Comment:
Do we need ot have this test if we have the test below? isn't it duplicating?

Reply:
Good point, they are basically doing the same thing, will removed the former and leave the latter